english/avatar/which-one.md

```diff
@@ -0,0 +1,6 @@
+You're the Avatar? Cool!
+
+Which one are you?
+
+[Aang]()
+[Korra]()
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.